client/src/components/ActivityList/ActivityList.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./ActivityList.css";
+
+// Class Based React Component
+class ActivityList extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "ActivityList__activity-list"
+    };
```

Comment:
You don't need to track the classList in state - just assign it to the `className` attribute